templates/components/common/header.html

```diff
@@ -21,8 +20,7 @@ <h1 class="header-logo header-logo--{{theme_settings.logo-position}}">
             {{> components/common/store-logo}}
         </div>
     {{/if}}
-
-
+    {{{region name="header_bottom"}}}
```

Comment:
By placement parameters. Placements require specifying a template file. We target a page with content based on both the region name AND the template file. 

So in the case of the home page it would target "pages/home". For the cart page - "pages/cart".